python/README.md

```diff
@@ -13,7 +15,7 @@ This lightweight Python client application connects to the Dremio Arrow Flight s
   - `python3 example.py -host '<DREMIO_HOST>' -user '<DREMIO_USERNAME>' -pass '<DREMIO_PASSWORD>'`
 
 ```
-usage: example.py [-h] [-host HOSTNAME] [-port FLIGHTPORT] -user USERNAME -pass PASSWORD [-query SQLQUERY] [-tls] [-certs TRUSTEDCERTIFICATES]
+usage: example.py [-h] [-host HOSTNAME] [-port FLIGHTPORT] -user USERNAME -pass PASSWORD [-query SQLQUERY] [-tls] [-certs TRUSTEDCERTIFICATES] [-sn SERVERNAME]
```

Comment:
Please pull SERVERNAME out into a separate PR.

Reply:
Removed SERVERNAME feature from this PR.